### ISSUE TITLE
Discourse - Non-text contrast issues

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#125]
**Dit probleem kan op meerdere pagina's aanwezig zijn, waaronder:**
<a href="https://community.developer.overheid.nl/t/officiele-bekendmakingen-binnenlandse-zaken-en-koninkrijksrelaties/726">Officiële bekendmakingen (Binnenlandse Zaken en Koninkrijksrelaties)</a>

**Probleembeschrijving:**
Probleem: Niet-tekstuele afbeeldingen (buttons) zoals de hart- en kettinglinkpictogrammen in de navigatie-element met class="post controls" hebben geen goed contrast.

Mogelijke verbeteringen: Verbeter het contrast van de pictogrammen door ze donkerder te maken. Als alternatief kun je de pictogrammen op contrasterende gekleurde cirkels of vierkanten plaatsen.

**Referentiecode:**
`<button class="widget-button btn-flat share no-text btn-icon" title="deel een link naar dit bericht"><svg class="fa d-icon d-icon-d-post-share svg-icon svg-node" aria-hidden="true"><use xlink:href="#link"></use></svg></button>`

**Gerelateerde WCAG:**
WCAG 1.4.11 Non-text contrast
Level AA

**Referentieafbeelding:**

![a contrast analyzing tool shows the contrast of the buttons shaped like a heart and a chain lack enough contrast with their background](https://github.com/user-attachments/assets/ee064941-285e-4c61-9c43-0c8c297a4e43)

_**Referentienummer:** A124_
